CMakeLists.txt

```diff
@@ -265,6 +270,11 @@ endif()
 
 ### INSTALLATION SECTION
 
+if (STARCACHE_SKIP_INSTALL)
```

Comment:
looks like it would be better moving following install scripts into a separate file e.g. `cmake/install.cmake`

and do a optional include by checking the SKIP_INSTALL flag. direct return sounds like misleading, if there are more lines added in future after the installation section.